packages/babel-register/src/browser.js

```diff
@@ -1,3 +1,5 @@
 // required to safely use babel/register within a browserify codebase
 
-export default function() {}
+export default function register() {}
+
+export function revert() {}
```

Comment:
do we need this?

Reply:
Yeah otherwise the interfaces aren't consistent. `browser.js` is swapped for `node.js` in browser builds, so they should both export the same interface. Seems like just an oversight.

I have no idea if anyone is _using_ that build, I can't imagine they are, but you never know :)